x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/RegressionIT.java

```diff
@@ -597,7 +596,7 @@ public void testAliasFields() throws Exception {
             fail("Failed to index data: " + bulkResponse.buildFailureMessage());
         }
 
-        long seed = randomLong();
+        long seed = 1000L; //fix seed
```

Comment:
Please can you add a comment that explains why the seed is fixed.  It would be good to say that we sometimes get outlier results, why that happens, and why it's OK.

Reply:
Thank you. I added an explanation comment.